webapp/src/containers/Rhs/__snapshots__/rhsHeader.test.tsx.snap

```diff
@@ -29,25 +29,23 @@ exports[`RHSHeader should match snapshot 1`] = `
           />
         </SVGWrapper>
       </IconButton>
-      <button
-        className="btn btn-primary share-record-btn false"
+      <IconButton
+        extraClass="false"
```

Comment:
can't we just not pass this prop if no `extraClass` needed? what is `extraClass="false"`?

Reply:
this is a snapshot